CMakeLists.txt

```diff
@@ -28,7 +28,20 @@ option(RE2C_BUILD_LIBS "Build libraries" OFF)
 option(RE2C_BUILD_RE2GO "Build re2go executable (an alias for `re2c --lang go`)" ON)
 
 # checks for programs
-find_package(BISON)
+if(UNIX)
+    find_program(
+      BISON_EXECUTABLE
+      NAMES bison
+      PATHS /usr /usr/local
+      PATH_SUFFIXES bin)
+elseif(WIN32)
+    find_program(
+      BISON_EXECUTABLE
+      NAMES bison.exe win_bison.exe
+      PATHS C:/
+      PATH_SUFFIXES "")
+endif()
+mark_as_advanced(BISON_EXECUTABLE)
```

Comment:
Why is this needed? From [the docs](https://cmake.org/cmake/help/latest/command/mark_as_advanced.html?highlight=s):

> An advanced variable will not be displayed in any of the cmake GUIs unless the show advanced option is on. In script mode, the advanced/non-advanced state has no effect.

So I think it affects only how the variable is displayed, but not the build process. In that case let's drop this (it's fine to have it displayed, and it is more important to keep the CMakeLists.txt as simple as possible).

Reply:
I tried to add the comment to line 44, but for some reason it shows lines 41-43 as well.